helios-core/src/main/kotlin/helios/instances/instancesEq.kt

```diff
@@ -48,38 +47,38 @@ interface JsNumberEqInstance : Eq<JsNumber> {
       is JsDecimal -> String.eq().run { this@eqv.value.toString().eqv(b.value) }
-      is JsLong -> Long.eq().run { this@eqv.value.eqv(b.value) }
-      is JsDouble -> Double.eq().run { this@eqv.value.toDouble().eqv(b.value) }
-      is JsFloat -> Float.eq().run { this@eqv.value.toFloat().eqv(b.value) }
-      is JsInt -> Long.eq().run { this@eqv.value.eqv(b.value.toLong()) }
+      is JsLong    -> Long.eq().run { this@eqv.value.eqv(b.value) }
+      is JsDouble  -> Double.eq().run { this@eqv.value.toDouble().eqv(b.value) }
+      is JsFloat   -> Float.eq().run { this@eqv.value.toFloat().eqv(b.value) }
+      is JsInt     -> Long.eq().run { this@eqv.value.eqv(b.value.toLong()) }
     }
-    is JsDouble -> when (b) {
+    is JsDouble  -> when (b) {
       is JsDecimal -> String.eq().run { this@eqv.value.toString().eqv(b.value) }
-      is JsLong -> Double.eq().run { this@eqv.value.eqv(b.value.toDouble()) }
-      is JsDouble -> Double.eq().run { this@eqv.value.eqv(b.value) }
-      is JsFloat -> Double.eq().run { this@eqv.value.eqv(b.value.toDouble()) }
-      is JsInt -> Double.eq().run { this@eqv.value.eqv(b.value.toDouble()) }
+      is JsLong    -> Double.eq().run { this@eqv.value.eqv(b.value.toDouble()) }
+      is JsDouble  -> Double.eq().run { this@eqv.value.eqv(b.value) }
+      is JsFloat   -> Double.eq().run { this@eqv.value.eqv(b.value.toDouble()) }
+      is JsInt     -> Double.eq().run { this@eqv.value.eqv(b.value.toDouble()) }
     }
-    is JsFloat -> when (b) {
+    is JsFloat   -> when (b) {
       is JsDecimal -> String.eq().run { this@eqv.value.toString().eqv(b.value) }
-      is JsLong -> Float.eq().run { this@eqv.value.eqv(b.value.toFloat()) }
-      is JsDouble -> Double.eq().run { this@eqv.value.toDouble().eqv(b.value) }
-      is JsFloat -> Float.eq().run { this@eqv.value.eqv(b.value) }
-      is JsInt -> Float.eq().run { this@eqv.value.eqv(b.value.toFloat()) }
+      is JsLong    -> Float.eq().run { this@eqv.value.eqv(b.value.toFloat()) }
+      is JsDouble  -> Double.eq().run { this@eqv.value.toDouble().eqv(b.value) }
+      is JsFloat   -> Float.eq().run { this@eqv.value.eqv(b.value) }
+      is JsInt     -> Float.eq().run { this@eqv.value.eqv(b.value.toFloat()) }
     }
-    is JsInt -> when (b) {
+    is JsInt     -> when (b) {
       is JsDecimal -> String.eq().run { this@eqv.value.toString().eqv(b.value) }
-      is JsLong -> Long.eq().run { this@eqv.value.toLong().eqv(b.value) }
-      is JsDouble -> Double.eq().run { this@eqv.value.toDouble().eqv(b.value) }
-      is JsFloat -> Float.eq().run { this@eqv.value.toFloat().eqv(b.value) }
-      is JsInt -> Int.eq().run { this@eqv.value.eqv(b.value) }
+      is JsLong    -> Long.eq().run { this@eqv.value.toLong().eqv(b.value) }
+      is JsDouble  -> Double.eq().run { this@eqv.value.toDouble().eqv(b.value) }
+      is JsFloat   -> Float.eq().run { this@eqv.value.toFloat().eqv(b.value) }
+      is JsInt     -> Int.eq().run { this@eqv.value.eqv(b.value) }
```

Comment:
Is this a setting in KtLint? I like it but it's something I haven't seen in Kotlin but only 

How can I add it in my own projects? :D

Reply:
No idea I think it was the detekt plugin from Intellij